app/src/main/java/org/simple/clinic/editpatient/EditPatientEffectHandler.kt

```diff
@@ -104,6 +113,17 @@ class EditPatientEffectHandler @AssistedInject constructor(
     }
   }
 
+  private fun fetchBpPassports(scheduler: Scheduler) =
+      ObservableTransformer<FetchBpPassportsEffect, EditPatientEvent> { fetchBpPassportsEffect ->
+        fetchBpPassportsEffect.subscribeOn(scheduler)
+            .flatMap { patientRepository.patientProfile(it.patientUuid) }
+            .filterAndUnwrapJust()
+            .map { patientProfile ->
+              patientProfile.withoutDeletedBusinessIds().businessIds.filter { it.identifier.type == BpPassport }
```

Comment:
You can add an `subscribeOn` here as well.

Reply:
Please reformat to 
```
    patientProfile
        .withoutDeletedBusinessIds()
        .businessIds
        .filter { it.identifier.type == BpPassport }
```

Reply:
Done